apps/web/src/utils/usernames.ts

```diff
@@ -469,6 +504,28 @@ export async function getBasenameAddress(username: BaseName) {
   } catch (error) {}
 }
 
+// Get username token `owner`
+export function buildBasenameOwnerContract(username: BaseName): ContractFunctionParameters {
+  const chain = getChainForBasename(username);
+  return {
+    abi: RegistryAbi,
+    address: USERNAME_BASE_REGISTRY_ADDRESSES[chain.id],
+    args: [namehash(username)],
+    functionName: 'owner',
+  };
+}
+
+export async function getBasenameOwner(username: BaseName) {
+  const chain = getChainForBasename(username);
+
+  try {
+    const client = getBasenamePublicClient(chain.id);
+    const owner = await client.readContract(buildBasenameOwnerContract(username));
+
+    return owner;
+  } catch (error) {}
```

Comment:
double checking we really want to silently fail here

Reply:
yeah it's fine in one-off utils for now I think.